### PR TITLE
Feature/encoding desc vizualization

### PR DESCRIPTION
**Pull request by ChiaraDipi**
First part of encoding description visualization. 
Elements handled in a specific way are
- Project description (`<projectDesc>`)
- Sampling declaration (`<samplingDecl>`)
- Editorial declaration (`<editorialDecl>`)
- Rendition (`<rendition>`)
- Namespace (`<namespace>`)
- Tags declaration (`<tagsDecl>`)

Other elements of encoding description are still rendered with generic element and will be handled in a separate PR

### CODE REVIEW

File: src/app/components/encoding-desc/encoding-desc.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
```
Comment (giacomocerre) [inline src/app/components/encoding-desc/encoding-desc.component.spec.ts:2]: Remove space

File: src/app/components/project-desc/project-desc.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
```
Comment (giacomocerre) [inline src/app/components/project-desc/project-desc.component.spec.ts:2]: Remove space

File: src/app/components/sampling-decl/sampling-decl.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
```
Comment (giacomocerre) [inline src/app/components/sampling-decl/sampling-decl.component.spec.ts:2]: Remove space

File: src/app/components/editorial-decl/editorial-decl.component.spec.ts [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
```
Comment (giacomocerre) [inline src/app/components/editorial-decl/editorial-decl.component.spec.ts:2]: Remove space

File: src/app/components/rendition/rendition.component.scss [hunk 5]
```diff
@@ -0,0 +1,23 @@
+.rendition {
+    margin-bottom: 10px;
+
+    &-section {
+        display: flex;
+        align-items: flex-start;
+        justify-content: flex-start;
+    }
+
```
Comment (giacomocerre) [inline src/app/components/rendition/rendition.component.scss:9]: Remove space

File: src/app/components/rendition/rendition.component.spec.ts [hunk 6]
```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
```
Comment (giacomocerre) [inline src/app/components/rendition/rendition.component.spec.ts:2]: Remove space

File: src/app/components/namespace/namespace.component.spec.ts [hunk 7]
```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
```
Comment (giacomocerre) [inline src/app/components/namespace/namespace.component.spec.ts:2]: Remove space

File: src/app/components/tags-decl/tags-decl.component.spec.ts [hunk 8]
```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
```
Comment (giacomocerre) [inline src/app/components/tags-decl/tags-decl.component.spec.ts:2]: Remove space

File: src/app/components/encoding-desc/encoding-desc.component.spec.ts (hunk 1 above)
Comment (giacomocerre) [inline src/app/components/encoding-desc/encoding-desc.component.spec.ts:2]: Remove space

File: src/app/components/project-desc/project-desc.component.spec.ts (hunk 2 above)
Comment (giacomocerre) [inline src/app/components/project-desc/project-desc.component.spec.ts:2]: Remove space

File: src/app/components/sampling-decl/sampling-decl.component.spec.ts (hunk 3 above)
Comment (giacomocerre) [inline src/app/components/sampling-decl/sampling-decl.component.spec.ts:2]: Remove space

File: src/app/components/editorial-decl/editorial-decl.component.spec.ts (hunk 4 above)
Comment (giacomocerre) [inline src/app/components/editorial-decl/editorial-decl.component.spec.ts:2]: Remove space

File: src/app/components/editorial-decl/editorial-decl.component.html [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+        <ng-container *ngTemplateOutlet="section; context: { label: 'hyphenation', items: data?.hyphenation, class: 'hyphenation' }"></ng-container>
+        <ng-container *ngTemplateOutlet="section; context: { label: 'segmentation', items: data?.segmentation, class: 'segmentation' }"></ng-container>
+        <ng-container *ngTemplateOutlet="section; context: { label: 'stdVals', items: data?.stdVals, class: 'stdVals' }"></ng-container>
+        <ng-container *ngTemplateOutlet="section; context: { label: 'interpretation', items: data?.interpretation, class: 'interpretation' }"></ng-container>
+    </ng-container>
+    <ng-container *ngSwitchDefault>
+        <evt-content-viewer *ngFor="let element of data?.content" [content]="element"></evt-content-viewer>
+    </ng-container>
+</ng-container>
+
```
Comment (giuliac89) [inline src/app/components/editorial-decl/editorial-decl.component.html:16]: Remove space

**Comment by giuliac89** [comment]
Resolve conflict

**Review by giuliac89 — CHANGES_REQUESTED** [review]

**Review by szenzaro — APPROVED** [review]

File: CHANGELOG.md [hunk 10]
```diff
@@ -10,6 +10,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Updated to Angular 9
 
 ### Added
+- Tags declaration visualization
+- Namespace declaration visualization
+- Rendition declaration visualization
+- Editorial declaration visualization
+- Sampling declaration visualization
+- Project description visualization
+- Encoding description visualization
```
[the remark this answers is not in the document]
Comment (ChiaraDipi) [reply CHANGELOG.md:19]: Next time

**Review by szenzaro — APPROVED** [review]

**Review by giuliac89 — APPROVED** [review]

**Pull request merged**